Rules/Rules.csproj

```diff
@@ -15,6 +15,12 @@
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.1' ">
+    <PackageReference Include="System.Reflection.TypeExtensions" Version="4.7.0" />
+    <PackageReference Include="Microsoft.Management.Infrastructure" Version="2.0.0" />
+    <Compile Remove="UseSingularNouns.cs" />
```

Comment:
I put this in the same block since the previous PR did the same, but I'd be happy to move it to its own block too